collect_app/src/main/res/values/strings.xml

```diff
@@ -106,7 +97,6 @@
 <string name="no_connection">No network connection available</string>
 <string name="no_items_display">Nothing available to display.</string>
 <string name="no_items_display_forms">Nothing available to display.\n\nTry getting and filling out a blank form.</string>
-<string name="no_items_display_sent_forms">Nothing available to display.\n\nTry sending finalized forms before viewing.</string>
```

Comment:
I think this actually should be used for when View Sent Form has nothing. Right now it uses `no_items_display_forms` which isn't right!